src/include/main/database.h

```diff
@@ -88,6 +94,7 @@ class Database {
     std::unique_ptr<transaction::TransactionManager> transactionManager;
     std::unique_ptr<storage::WAL> wal;
     std::shared_ptr<spdlog::logger> logger;
+    std::unique_ptr<common::FileInfo> lockStatus;
```

Comment:
is it ok? I need to hold the `.lock` file information after I open it. otherwise the lock will lose its efficacy